src/modules/functions.js

```diff
@@ -0,0 +1,164 @@
+import menuImgSrc from '../menu.png';
+import enterImgSrc from '../enter.png';
+import refreshImgSrc from '../refresh.png';
+import trashImgSrc from '../trash.png';
+import { addCheckboxListenerOnLoad } from './check_box.js';
+
+export const createFrame = () => {
+  const h1Container = document.querySelector('.h1-container');
+  const refreshImg = document.createElement('img');
+  const container = document.querySelector('.to-do-list');
+  refreshImg.src = refreshImgSrc;
+  refreshImg.className = 'refresh-img';
+  h1Container.appendChild(refreshImg);
+  const form = document.createElement('form');
+  const input = document.createElement('input');
+  input.className = 'add-task-input';
+  input.type = 'text';
+  input.name = 'description';
+  input.placeholder = 'Add to your list...';
+  const inputLine = document.createElement('div');
+  inputLine.className = 'input-line';
+  inputLine.appendChild(input);
+  const enterImg = document.createElement('img');
+  enterImg.src = enterImgSrc;
+  enterImg.className = 'enter-img';
+
+  inputLine.appendChild(enterImg);
+  form.appendChild(inputLine);
+  container.appendChild(form);
+};
+
+export const addLocalStorage = () => {
+  const ul = document.createElement('ul');
+  const arrayOfTasks = [];
+  const storedTasks = JSON.parse(localStorage.getItem('tasks'));
+  for (let i = 0; i < storedTasks.length; i += 1) {
+    const li = document.createElement('li');
+    const checkBox = document.createElement('input');
+    checkBox.type = 'checkbox';
+    const content = document.createElement('div');
+    content.className = 'content';
+    content.appendChild(checkBox);
+    checkBox.checked = storedTasks[i].completed;
+    const inp = document.createElement('input');
+    inp.value = storedTasks[i].description;
+    inp.type = 'text';
+    inp.classList = ('added-task uneditable');
+    if (checkBox.checked) {
+      inp.classList.add('line-through');
+    }
```

Comment:
I guess using the forEach (ES6 syntax) method here makes your code more professional.